### PR TITLE
Add exceptions for PCViewerCommand error handling

### PR DESCRIPTION
**Pull request by Reynold-SL**
-Create exceptions for PCViewerCommand Error Handling
-Reformat code documentation to comply with checkstyle

### CODE REVIEW

File: src/main/java/seedu/bigpp/command/viewercommand/ViewerEditCommand.java [hunk 1]
```diff
@@ -10,12 +13,20 @@ public ViewerEditCommand(String arguments) {
...
-    public String executeCommand() {
+    public String executeCommand() throws PPException {
+        // throw exception if no index is selected
+        if (super.getArguments().equals("")) {
+            throw new ViewerMissingIndexException();
+        }
         int pcIndex = Integer.parseInt(super.getArguments()) - 1;
+        // throw exception if index selected is out of the PCList range
+        if (pcIndex < 0 || Integer.parseInt(super.getArguments()) > PCList.getList().size()) {
+            throw new ViewerInvalidIndexException();
```
Comment (Ryujikjs) [inline src/main/java/seedu/bigpp/command/viewercommand/ViewerEditCommand.java:28]: please use PPIndexOutOfboundsException

File: src/main/java/seedu/bigpp/command/viewercommand/ViewerViewCommand.java [hunk 2]
```diff
@@ -15,8 +18,16 @@ public ViewerViewCommand(String arguments) {
...
-    public String executeCommand() {
+    public String executeCommand() throws PPException {
         String index = super.getArguments();
+        // throw exception if no index is selected
+        if (index.equals("")) {
+            throw new ViewerMissingIndexException();
+        }
+        // throw exception if index selected is out of the PCList range
+        if (Integer.parseInt(index) < 0 || Integer.parseInt(super.getArguments()) > PCList.getList().size()) {
+            throw new ViewerInvalidIndexException();
```
Comment (Ryujikjs) [inline src/main/java/seedu/bigpp/command/viewercommand/ViewerViewCommand.java:29]: please use PPIndexOutOfboundsException

**Review by Ryujikjs — CHANGES_REQUESTED** [review]
Can make the required changes to use PPIndexOutOfBoundsException

File: src/main/java/seedu/bigpp/command/viewercommand/ViewerViewCommand.java (hunk 2 above)
Comment (Reynold-SL) [reply src/main/java/seedu/bigpp/command/viewercommand/ViewerViewCommand.java]: Code update using PPIndexOutofboundsException

File: src/main/java/seedu/bigpp/command/viewercommand/ViewerEditCommand.java (hunk 1 above)
Comment (Reynold-SL) [reply src/main/java/seedu/bigpp/command/viewercommand/ViewerEditCommand.java]: Code update using PPIndexOutofboundsException

**Review by Ryujikjs — APPROVED** [review]
Looks good to me

**Pull request merged**